src/renderer/components/ChangelogModal/index.tsx

```diff
@@ -33,6 +37,15 @@ function hidechangelog() {
         showchangelog
     } });
 }
+function versionchanged() {
```

Comment:
`versionChanged`, also it should be a const function component `const versionChanged = () => {`